src/components/HomeContainer/Talent/Talent.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import Typography from "../../utility/Typography/Typography";
+import styles from "./Talent.module.css"
+import "../../../App.css";
+
+const Talent = () => {
+  return (
+    <section className={styles.container}>
+      <div className={styles.text_wrapper}>
+        <div className={styles.heading}>
+          <Typography elType="h1Large">
+            Find the<br/> best <span>talent</span>
+          </Typography>
+        </div>
+        <div className={styles.content}>
+          <div className={styles.rectangle}>
```

Comment:
Why do we need this rectangle div?

Reply:
The figma design for Desktop devices has this "rapture blue" rectangle shape (right above the body text). I didn't see an image asset for it, so I created it.
![Screenshot 2024-07-12 at 8 38 10 AM](https://github.com/user-attachments/assets/292b5122-0dce-4760-94df-1423ebc920e1)